File: Tl2.2 SOAP_DOTNET_EUREKA_BANK_GR08/02.CLIMOVIL/app/src/main/java/ec/edu/monster/model/CuentaModel.java
package ec.edu.monster.model;

public class CuentaModel {
    private double decCuenSaldo;
    private int intCuenContMov;
    private String strCuenNumero;

    public CuentaModel() {
    }

    public CuentaModel(double decCuenSaldo, int intCuenContMov, String strCuenNumero) {
        this.decCuenSaldo = decCuenSaldo;
        this.intCuenContMov = intCuenContMov;
        this.strCuenNumero = strCuenNumero;
    }

    public double getDecCuenSaldo() {
        return decCuenSaldo;
    }

    public void setDecCuenSaldo(double decCuenSaldo) {
        this.decCuenSaldo = decCuenSaldo;
    }

    public int getIntCuenContMov() {
        return intCuenContMov;
    }

    public void setIntCuenContMov(int intCuenContMov) {
        this.intCuenContMov = intCuenContMov;
    }

    public String getStrCuenNumero() {
        return strCuenNumero;
    }

    public void setStrCuenNumero(String strCuenNumero) {
        this.strCuenNumero = strCuenNumero;
    }
}

File: Tl2.4 RESTFULL_DOTNET_EUREKA_BANK_GR08/02.CLIWEB/CLIWEB_EUREKA_GR08/ec.edu.monster.model/CuentaModel.cs
using System;

namespace CLIWEB_EUREKA_GR08.ec.edu.monster.model
{
    public class CuentaModel
    {
        public string ChrCuenCodigo { get; set; }
        public string ChrMoneCodigo { get; set; }
        public string ChrSucucodigo { get; set; }
        public string ChrEmplCreaCuenta { get; set; }
        public string ChrClieCodigo { get; set; }
        public double DecCuenSaldo { get; set; }
        public DateTime? DttCuenFechaCreacion { get; set; }
        public string VchCuenEstado { get; set; }
        public int IntCuenContMov { get; set; }
        public string ChrCuenClave { get; set; }
    }
}

